docs/src/main/sphinx/connector/vertica.md

```diff
@@ -0,0 +1,234 @@
+  - `DATE`
+  -
+:::
+
+No other types are supported.
+
+Unsupported Vertica types can be converted to `VARCHAR` with the
+`vertica.unsupported_type_handling` session property. The default value for
+this property is `IGNORE`.
+
+```sql
+SET SESSION vertica.unsupported_type_handling='CONVERT_TO_VARCHAR';
+```
+
+### Trino to Vertica type mapping
+
+The connector maps Trino types to the corresponding Vertica types according to
+the following table:
+
+:::{list-table} Trino to Vertica type mapping
+:widths: 50, 50
+:header-rows: 1
+
+* - Trino type
+  - Vertica type
+* - `BOOLEAN`
+  - `BOOLEAN`
+* - `TINYINT`
+  - `BIGINT`
+* - `SMALLINT`
+  - `BIGINT`
+* - `INTEGER`
+  - `BIGINT`
+* - `BIGINT`
+  - `BIGINT`
+* - `REAL`
+  - `DOUBLE PRECISION`
+* - `DOUBLE`
+  - `DOUBLE PRECISION`
+* - `DECIMAL(p, s)`
+  - `DECIMAL(p, s)`
+* - `CHAR`
+  - `CHAR`
+* - `VARCHAR`
+  - `VARCHAR`
+* - `VARBINARY`
+  - `VARBINARY`
+* - `DATE`
+  - `DATE`
+:::
+
+No other types are supported.
+
+```{include} jdbc-type-mapping.fragment
+```
+
+(vertica-sql-support)=
+## SQL support
+
+The connector provides read and write access to data and metadata in Vertica. In
+addition to the [globally available](sql-globally-available) and [read
+operation](sql-read-operations) statements, the connector supports the following
+features:
+
+- [](sql-data-management)
+- [](/sql/create-table)
+- [](/sql/create-table-as)
+- [](/sql/drop-table)
+- [](/sql/alter-table) excluding `DROP COLUMN`
+- [](/sql/create-schema)
+- [](/sql/drop-schema)
+
+```{include} alter-table-limitation.fragment
+```
+
+## Table functions
+
+The connector provides specific [table functions](/functions/table) to
+access Vertica.
+
+(vertica-query-function)=
+### `query(VARCHAR) -> table`
+
+The `query` function allows you to query the underlying database directly. It
+requires syntax native to the data source, because the full query is pushed down
+and processed in the data source. This can be useful for accessing native
+features or for improving query performance in situations where running a query
+natively may be faster.
+
+The `query` table function is available in the `system` schema of any
+catalog that uses the Vertica connector, such as `example`. The
+following example passes `myQuery` to the data source. `myQuery` has to be a
+valid query for the data source, and is required to return a table as a result:
+
+```sql
+SELECT
+  *
+FROM
+  TABLE(
+    example.system.query(
+      query => 'myQuery'
+    )
+  );
+```
+
+```{include} query-table-function-ordering.fragment
+```
+
+## Performance
+
+The connector includes a number of performance features, detailed in the
+following sections.
+
+### Pushdown
+
+The connector supports pushdown for a number of operations:
+
+- [](join-pushdown)
+- [](limit-pushdown)
+
+```{include} join-pushdown-enabled-false.fragment
+```
+
+### Table statistics
+
+You can use [](/sql/analyze) statements in Trino to populate the table
+statistics in Vertica. The [cost-based
```

Comment:
This connector doesn't support analyze statement. Please remove this sentence.

Reply:
Will send

Reply:
https://github.com/trinodb/trino/pull/23983